### ISSUE TITLE
test model and train model are different?

### ISSUE THREAD

**Issue by Stonebobo** [kuihua/MSPFN#4]
I carefully checked the generator function in the MSPFN.py file and the generator function in the TEST_MSPFN.py file, and I found that they are different. Does it mean that the model trained in the training set is not the same as the model in the test set? E.g
In TEST_MSPFN.py, line 183, with tf.variable_scope('BCM_{}'.format(n)):
But there is no BCM_{} in MSPFN, the corresponding is with tf.variable_scope('URAB_{}'.format(n)) in MSPFN:

**Comment by kuihua** [comment]
I am sorry for this problem. The TEST_MSPFN.py is the abalation version, and i have uploaded the revised version. You can try it again. Best wishes!

**Comment by Stonebobo** [comment]
OK, it is working normally, but the effect is not very good. Since I cannot download dataset from Google, can you put your dataset in Baidu Cloud? Then give me a link, thank you!

**Comment by kuihua** [comment]
ok, you can download the training datasets (.npy) from the following link:
链接：https://pan.baidu.com/s/1e3jjUeFvaS6-nrZzXIdAQg 
提取码：vp1g

**Comment by Stonebobo** [comment]
thank you for your help！
